infrastructure/prometheus/alarm/prometheus.yml

```
@@ -0,0 +1,16 @@
+rule_files:
+  - "/etc/prometheus/rules/*.yml"
+
+scrape_configs:
+  - job_name: 'prometheus-alarm'
+    static_configs:
+      - targets: [ 'localhost:9094' ]
+
+  - job_name: 'alarm-service'
+    metrics_path: '/actuator/prometheus'
+    static_configs:
+      - targets: [ 'host.docker.localhost:10054' ]
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)